pages/login/LoginHandler.inc.php

```diff
@@ -245,11 +245,11 @@ function resetPassword($args, $request) {
 			import('lib.pkp.classes.mail.MailTemplate');
 			$mail = new MailTemplate('PASSWORD_RESET');
 			$this->_setMailFrom($request, $mail, $site);
-			$mail->assignParams(array(
-				'username' => $user->getUsername(),
-				'password' => $newPassword,
-				'siteTitle' => $site->getLocalizedTitle()
-			));
+			$mail->assignParams([
+				'username' => htmlspecialchars($user->getUsername()),
+				'password' => htmlspecialchars($newPassword),
```

Comment:
In general, this issue is about adding escaping to places where it's missing -- if a password happened to include `<i>`, for example, it'd be interpreted as HTML. The addition of `htmlspecialchars` fixes this. If you think I've missed something particular to passwords, please let me know -- but I'm confident that this solves problems rather than introducing them.